src/main/kotlin/io/github/rybalkinsd/kohttp/ext/RequestExt.kt

```diff
@@ -0,0 +1,51 @@
+package io.github.rybalkinsd.kohttp.ext
+
+import okhttp3.Request
+import okio.Buffer
+import java.nio.charset.Charset
+
+/**
+ *  Build curl command of the request
+ *
+ * @author doyaaaaaken
+ */
+internal fun Request.buildCurlCommand(): String {
+    var compressed = false
+    //TODO: test
```

Comment:
I write test method sample in RequestExtTest.kt.
But not yet write test in many case.
(I write test only about simple GET request case)

After I confirm whether this PR's direction is correct, I'll write test code.